src/Models/User.php

```diff
@@ -13,7 +14,7 @@
  * @property string $email
  * @property string $sub
  */
-class User extends Authenticatable
+class User extends Authenticatable implements StatefulUser
 {
     use HasFactory;
```

Comment:
The user model must implement StatefulUser, which is a requirement from the library. See quickstart example:
https://github.com/auth0-samples/auth0-laravel-php-web-app/blob/master/app/app/Models/User.php#L13